packages/ra-ui-materialui/src/input/ReferenceInput.tsx

```diff
@@ -283,6 +283,7 @@ export const ReferenceInputView: FunctionComponent<ReferenceInputViewProps> = ({
         setFilter,
         setPagination,
         setSort,
+        helperText,
```

Comment:
Reference inputs (all of them) should not accept an `helperText`. Their child might though. I think we should warn the user when they set the `helperText` prop on any of the reference inputs and explain they should add it on the child instead.

Reply:
ok with it